### ISSUE TITLE
"is not defined in object type" error with marketplace action

### ISSUE THREAD

**Issue by frank-long** [rhysd/actionlint#18]
I'm using dorny/paths-filter and that sets a dynamic output for me, but actionlint is complaining about the output not being there, see code below. I can't see how I can fix that.

```
- name: Checkout repo
        uses: actions/checkout@v2
      - id: filterchanges
        uses: dorny/paths-filter@v2
        with:
          filters: 'files: .github/workflows/*.yml'
      - name: lint
        if: ${{ steps.filterchanges.outputs.files == 'true'}}
[actionlint] reported by reviewdog 🐶 property "files" is not defined in object type {changes: any} [expression]


```

**Comment by frank-long** [comment]
not an issue, closing

**Comment by frank-long** [comment]
If I change to: ${{ steps.filterchanges.outputs.changes.files == 'true'}} then it won't complain, but it won't run the code because it won't evaluate to true: see https://github.com/dorny/paths-filter. Something is wrong the actionlint I suspect

**Comment by rhysd** [comment]
It's because the `files` output is not defined in `action.yml`. Is it really correct output name?

https://github.com/dorny/paths-filter/blob/v2/action.yml#L47-L49

**Comment by rhysd** [comment]
I'm not familiar with the action, but it seems to set its outputs dynamically. So it looks not possible to check them statically.

**Comment by frank-long** [comment]
Yes they are set dynamically in that action. Is there a way to disable this check on this line so I can still run without reporting a failure in my CI?

**Comment by rhysd** [comment]
Currently no way. I'll fix this false positive on next patch update.

**Comment by rayray** [comment]
@rhysd Apologies if this is already documented somewhere, but can you share how to set the `skip_outputs` flag from CLI? Or config file?

**Comment by ventsyv** [comment]
When I run actionlint, I get 

```
property "jf_docker_url" is not defined in object type {actions_runner_debug: string; actions_step_debug: string; github_token: string; jf_access_token: string; jf_base_url: string; jf_url: string} [expression]
    |
115 |           jf_url: ${{ secrets.JF_DOCKER_URL }}
```

Obviously there is a repo secret defined with that name.

**Comment by fairclothjm** [comment]
Hi, I am getting this error and I can't get the `skip_outputs` setting work properly ignore outputs. Are there any plans to extend the actionlint config file to ignore jobs/steps or take the `ignore` regex?
